test/controller/bankrupt-officer.spec.ts

```diff
@@ -22,7 +26,7 @@ const mockResponse = () => {
   res.status = sinon.stub().returns(res);
   return res as unknown as Response;
 };
-const req = { } as unknown as Request;
+const req = { session: getSessionRequest() } as unknown as Request;
```

Comment:
What does the `as unknown as Request` stands for here? Why is it set as both unknown and Request?

Reply:
Thanks Yiannis. I am using it, and also advised by the TS compiler on your VSCode, to cast our test custom object to a primitive type, in this context Request. But after the change that I have done it can be removed. So I am going through the cleaning of the `unknown` keywords :)

Reply:
If they are being cleaned now, may as well remove it in this update then.

Reply:
Thanks! Removed all the unnecessary `unknown` conversion type :)